Added a tip to autocomplete styles using values (#2996)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/css/reference.md

1. Enter a valid value for that property and press **Enter**.  In the **DOM Tree**, a `style` attribute has been added to the element.

> > [!TIP]
> > Alternatively, you can also enter the value in the property field, and DevTools will suggest a list of matching **property: value** pairs to select from. For example, if you enter `bold` in the property field then DevTools will suggest `font-weight: bold` and `font-weight: bolder` as the possible rules. Press **Enter** to apply the rule.

Reviewer: ```suggestion
Alternatively, enter the value in the property field, and DevTools will then suggest a list of matching **property: value** pairs to select from. For example, if you enter `bold` in the property field, DevTools suggests `font-weight: bold` and `font-weight: bolder` as the possible rules. Press **Enter** to apply the rule.
```
* prefer present not future tense, per style guide
* Tip is deprecated per https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#alerts-note-tip-important-caution-warning
* I'll test this UI, read this article section, and possible enter a later rewrite comment.
* Omit "then" in an "If" sentence, per https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/if-vs-whether-vs-when
